### ISSUE TITLE
Remove system property to check if OJDK MHs are enabled

### ISSUE THREAD

**Issue by babsingh** [eclipse-openj9/openj9#12811]
Once OJDK MHs are enabled everywhere, the following commit in #12779 should be reverted: **Add a system property to check if OJDK MHs are enabled**.

Also, the following TKG PR should be reverted: https://github.com/adoptium/TKG/pull/198.

**Comment by pshipton** [comment]
I don't think we should ship a release containing this system property, so if we enable OJDK MH version by version across multiple releases, we can fix the system property so it's only defined where necessary.
